src/main/java/com/minecolonies/colony/buildings/AbstractBuilding.java

```diff
@@ -482,6 +484,7 @@ private void requestWorkOrder(int level)
         }
 
         colony.getWorkManager().addWorkOrder(new WorkOrderBuild(this, level));
+        LanguageHandler.sendPlayersLocalizedMessage(EntityUtils.getPlayersFromUUID(colony.getWorld(), colony.getPermissions().getMessagePlayers()), "com.minecolonies.workOrderAdded");
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Split this 183 characters long line (which is greater than 180 authorized). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS00103)